Documentation/gettingstarted/clustermesh/clustermesh.rst

```diff
@@ -26,6 +26,39 @@ Cluster Addressing Requirements
 * The network between clusters must allow the inter-cluster communication. The
   exact ports are documented in the :ref:`firewall_requirements` section.
 
+Cilium installation may not default to a tunneled datapath mode in a specific
+cloud environment. This can be overridden with:
+
+ * ConfigMap option ``tunnel=vxlan``
+ * Helm option ``--set tunnel=vxlan``
+ * ``cilium install`` option ``--config tunnel=vxlan``
```

Comment:
Nitpick: The indent on this list items marks them as a “quote” (they're put in `<blockquote>` HTML tags)
```suggestion
* ConfigMap option ``tunnel=vxlan``
* Helm option ``--set tunnel=vxlan``
* ``cilium install`` option ``--config tunnel=vxlan``
```

Reply:
Done, thanks!